README.rst

```diff
@@ -2,8 +2,17 @@
                             Tarantool C client libraries
 -------------------------------------------------------------------------------
 
-.. image:: https://travis-ci.org/tarantool/tarantool-c.svg?branch=master
-    :target: https://travis-ci.org/tarantool/tarantool-c
+.. image:: https://img.shields.io/github/workflow/status/tarantool/tarantool-c/fast_testing/master?label=Linux%20test
```

Comment:
TBH, I like 'testing' more than 'test' in this context. It is status of the testing, not a test.

Reply:
Fixed.